Doc/library/imaplib.rst

```diff
@@ -55,6 +55,9 @@ base class:
    .. versionchanged:: 3.9
       The optional *timeout* parameter was added.
 
+   .. versionadded:: 3.9
+      :meth:`IMAP4.unselect` was added.
```

Comment:
This markup should be removed. ".. versionadded:: 3.9" in the method doc is enough.